l/lotusscript.lss

```diff
@@ -0,0 +1,3 @@
+ Sub Click(Source As Button)
```

Comment:
Does the statement need to be enclosed in this function or can the Print statement be usable on its own?

Reply:
It has to be enclosed in the function. Otherwise I don't think there's another way to trigger the script.

Reply:
In that case, we should run it on initialisation.